motion_tracker/utils/crop_vis.py

```diff
@@ -41,3 +41,18 @@ def show_stages_of_random_crop(img, box_coords, output_width=256, output_height=
                                                   output_width, output_height)
     add_box(final_img, box_coords)
     show_img("Random Crop", final_img)
+
+def show_single_stage(img, box_coords, title="Happy Dance Image"): 
+    """Show an image with its surrounding bounding box
+
+    Args: 
+    ----
+        img: np.ndarray
+        box_coords: np.ndarray
+    """
+    
+    bb_coords = Coords(box_coords[0], box_coords[1], 
+                box_coords[2], box_coords[3])
+    img_w_bb = add_box(img, bb_coords)
+    show_img(title, img_w_bb)
+
```

Comment:
This is used at the bottom of the `image_generator.py` to give a kind of slideshow of random croppings for a given image.